src/main/app/claims/calendarClient.ts

```diff
@@ -0,0 +1,18 @@
+import { request } from 'client/request'
+import { claimApiBaseUrl } from 'claims/claimStoreClient'
+import { MomentFormatter } from 'utils/momentFormatter'
+import { Moment } from 'moment'
+import { MomentFactory } from 'shared/momentFactory'
+
+export class CalendarClient {
+
+  static getNextWorkingDay (date: Moment, addDays: number = 0): Promise<Moment> {
+    const url: string = `${claimApiBaseUrl}/calendar/next-working-day`
```

Comment:
This doesn't need to be constructed each time the method is invoked; move `url` to the class itself